### PR TITLE
POC: Hydra + Oathkeeper

### PR DESCRIPTION
**Pull request by jakkab**
**Description**

Changes proposed in this pull request:

- Add Hydra chart

**Related issue(s)**
#3399

### CODE REVIEW

File: resources/ory/charts/oathkeeper/templates/api-server.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-api-server
+  namespace: {{ .Release.Namespace }}
+  labels:
+    kyma-project.io/installation: ""
```
Comment (piotrmsc) [inline resources/ory/charts/oathkeeper/templates/api-server.yaml]: is this label required?

**Comment by piotrmsc** [comment]
In the first iteration, let's leave oathkeeper apiserver exposed like it is right now, we will think of using only one host in the future

File: resources/ory/charts/hydra/templates/oauth2.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      containers:
+        - name: {{ .Chart.Name }}-oauth2-server
+          image: oryd/hydra:v1.0.0-rc.8
+          imagePullPolicy: IfNotPresent
+          command:
+            - "hydra"
+          args:
+            - "serve"
+            - "all"
+            - "--dangerous-force-http"
```
[the remark this answers is not in the document]
Comment (jakkab) [reply resources/ory/charts/hydra/templates/oauth2.yaml]: We agreed to leave the flag for now.

**Pull request merged**